hikari/api/entity_factory.py

```diff
@@ -54,21 +54,21 @@
 
 
 @attr_extensions.with_copy
-@attr.s(slots=True, weakref_slot=False)
+@attr.define(weakref_slot=False)
 class GatewayGuildDefinition:
     """A structure for handling entities within guild create and update events."""
 
-    guild: guild_models.GatewayGuild = attr.ib()
+    guild: guild_models.GatewayGuild
     """Object of the guild the definition is for."""
 
-    channels: typing.Optional[typing.Mapping[snowflakes.Snowflake, channel_models.GuildChannel]] = attr.ib()
+    channels: typing.Optional[typing.Mapping[snowflakes.Snowflake, channel_models.GuildChannel]]
     """Mapping of channel IDs to the channels that belong to the guild.
 
     Will be `builtins.None` when returned by guild update gateway events rather
     than create.
     """
 
-    members: typing.Optional[typing.Mapping[snowflakes.Snowflake, guild_models.Member]] = attr.ib()
+    members: typing.Optional[typing.Mapping[snowflakes.Snowflake, guild_models.Member]]
```

Comment:
Seems a bit inconsistent to reintroduce this behaviour where we arbitrarily do or don't use auto_attribs in random places

Reply:
LGTM otherwise